uamqp/error.py

```diff
@@ -159,6 +288,14 @@ class AMQPConnectionError(AMQPException):
     """
 
 
+class ConnectionClose(AMQPConnectionError):
+    pass
+
+
+class VendorConnectionClose(ConnectionClose):
```

Comment:
As mentioned above - let's clean out this and VendorLinkDetach

Reply:
removed